arviz/plots/distplot.py

```diff
@@ -90,6 +90,48 @@ def plot_dist(
     Returns
     -------
     ax : matplotlib axes
+
+    Examples
+    --------
+    Plot an integer distribution
+
+    .. plot::
+        :context: close-figs
+
+        >>> import matplotlib.pyplot as plt
```

Comment:
Being very picky, this line is actually not used, otherwise it is completely spotless

Reply:
Ah good catch. @jscarbor could you delete just this one line? After that this will be ready to merge.

Reply:
Good catch indeed @OriolAbril . Done. And I confirmed that plot_dist.py where I copied those import from does in fact utilize the import.